magento/adobe-stock-integration
# fix color filter test

… by entering a search term to nullify default gallery





### Description (*)

Fixes the stock color filter test's asserting on empty results.

### Fixed Issues (if relevant)

1. #745 

### Manual testing scenarios (*)

1. Run the filter test and see it pass! `vendor/bin/mftf run:test AdminAdobeStockColorFilterTest --remove --verbose`
